foundry.toml

```diff
@@ -8,9 +8,7 @@ libs = [
 ]
 
 remappings = [
-    "forge-std/=lib/forge-std/src/",
```

Comment:
we still need the remappings in foundry.toml. Craig removed the `remappings.txt` file so thats the main source now

Reply:
done